src/main/java/xbot/common/subsystems/vision/AprilTagVisionIOPhotonVisionSimulated.java

```diff
@@ -64,6 +64,9 @@ public AprilTagVisionIOPhotonVisionSimulated(@Assisted String name, @Assisted Tr
         // Add sim camera
         var cameraProperties = new SimCameraProperties();
         cameraSim = new PhotonCameraSim(camera, cameraProperties);
+        cameraSim.enableRawStream(true);
+        cameraSim.enableProcessedStream(true);
+        cameraSim.enableDrawWireframe(true);
```

Comment:
It's possible this will prove too resource intensive on slower machines and then we can come up with more of a dynamic, preferences based config. But in the short-run I think this will help with debugging.